README.md

```diff
@@ -33,7 +59,7 @@ seems to be a solid approach:
 
 ```bash
 
-rm -rf web/profiles/contrib/ web/modules/contrib/ web/themes/contrib/;
+rm -rf web/;
```

Comment:
Hi Finn,
Any chance you can get a second opinion on this from Steve who I wrote it first?  I don't know why but I am feeling a bit uncomfortable wiping out the entire "web/" where we may have custom modules, settings files, etc. :(

Reply:
Second on this, it seems odd to say rm the whole web directory.

Reply:
Agreed - it's a bit of an assumption. It does work if you're just installing it all from composer for dev purposes and blowing it away repeatedly, but I agree this is not an assumption we necessarily want to make.